shell/shared/demo.js

```diff
@@ -84,7 +84,21 @@ if (Meteor.isServer) {
         // Log them in on this connection.
         return Accounts._loginMethod(this, "createDemoUser", arguments,
             "demo", function () { return { userId: userId }; });
+      },
+
+      getAppNameFromAppId: function(packageId) {
```

Comment:
Interesting. I'm not sure I fully understand how I'd implement this but am tentatively enthusiastic.